spaces/S000166/properties/P000090.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000166
+property: P000090
+value: true
+---
+
+By definition.
```

Comment:
```suggestion
The Alexandrov topology on a preordered set is Alexandrov.
```

'By definition' is morally true but not technically true, right? 
Not sure if this is worth worrying about. What do you think? Feel free to reject of course.

Reply:
I think this one is fine, given the fourth equivalent definition of [P90](https://topology.pi-base.org/properties/P000090).